build.gradle.kts

```
@@ -383,7 +383,7 @@ subprojects {
         }
 
         // Set Gradle property 'au.com.shiftyjelly.pocketcasts.leakcanary=true' to enable Leak Canary.
-        // You can do this using the local.properties file or any other avaialable Gradle mechanism.
+        // You can do this using the gradle.properties file or any other avaialable Gradle mechanism.
```

```suggestion
        // You can do this using the gradle.properties file or any other available Gradle mechanism.
```
It's not related to this PR but there is a minor typo.

This is a really handy way to turn on leak canary, much easier than it used to be, very nice!